src/pages/registration/references_tab/components/CorrigendumForField.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { FC, useState } from 'react';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { Typography } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import Truncate from 'react-truncate';
+import { Registration } from '../../../../types/registration.types';
+import { autocompleteTranslationProps } from '../../../../themes/mainTheme';
+import useSearchRegistrations from '../../../../utils/hooks/useSearchRegistrations';
+import { ReferenceFieldNames } from '../../../../types/publicationFieldNames';
+import { AutocompleteTextField } from '../../../../components/AutocompleteTextField';
+import EmphasizeSubstring from '../../../../components/EmphasizeSubstring';
+import { StyledFlexColumn } from '../../../../components/styled/Wrappers';
+import { JournalPublicationInstance } from '../../../../types/publication_types/journalRegistration.types';
+import { displayDate } from '../../../../utils/date-helpers';
+import useDebounce from '../../../../utils/hooks/useDebounce';
+import { API_URL } from '../../../../utils/constants';
+import JournalField from './JournalField';
+
+const typeFilter = `(
+entityDescription.reference.publicationInstance="JournalArticle" 
+OR 
+entityDescription.reference.publicationInstance="JournalShortCommunication"
+)`;
+
+const CorrigendumForField: FC = () => {
+  const { t } = useTranslation('registration');
+  const { values, setFieldValue, setFieldTouched } = useFormikContext<Registration>();
+  const [searchTerm, setSearchTerm] = useState('');
+  const debouncedSearchTerm = useDebounce(searchTerm);
+  const [journalRegistrationsSearch, isLoadingRegistrationsSearch] = useSearchRegistrations(
+    `${typeFilter} AND *${debouncedSearchTerm}*`
```

Comment:
wow. dette kan ikke være bra 😅 - kan vi søke på hva som helst? kanskje tom DROP table xx? @elkebab

Reply:
Vi kan søke på alle properties på en Registration ja 
Dette er en ElasticSearch feature som jeg har skjønt det, så håper og tror det er innbakt håndtering av drop table osv der 😬

Reply:
Ingen fare for sql-injection siden DynamoDB er en nosql-base. Og ElasticSearch skal kunne håndtere sånne ting, ja.